zebra/zebra_netns_notify.c

```diff
@@ -41,6 +41,10 @@
 #define ZEBRA_NS_POLLING_INTERVAL_MSEC     1000
 #define ZEBRA_NS_POLLING_MAX_RETRIES  200
 
+#if !defined(__GLIBC__)
+#define basename(src) (strrchr(src,'/') ? strrchr(src,'/')+1 : src)
```

Comment:
Just loudly thinking, if we need to add an additional check here:

```c
#ifndef basename
    #define basename(src) (strrchr(src,'/') ? strrchr(src,'/')+1 : src)
#endif
```

Reply:
what would that get us ? it is in a .c source file unless its included multiple times, it wont be useful.